frontend/package.json

```diff
@@ -17,18 +17,15 @@
     "e2e-test": "playwright test tests/console/ -c playwright.config.ts",
     "e2e-test-enterprise": "playwright test tests/console-enterprise/ -c playwright.enterprise.config.ts",
     "e2e-test:ui": "playwright test tests/console/ -c playwright.config.ts --ui",
-    "test": "TZ=GMT vitest run",
-    "test:coverage": "TZ=GMT vitest run --coverage",
-    "test:ui": "TZ=GMT vitest --ui",
-    "test:watch": "TZ=GMT vitest --watch --ui",
+    "test": "TZ=GMT bun test --coverage --concurrent src/**/*.test.{ts,tsx} src/components/**/*.test.{ts,tsx} src/components/pages/**/*.test.{ts,tsx} src/components/pages/topics/**/*.test.{ts,tsx}",
     "type:check": "tsc --noEmit --incremental --p tsconfig.base.json",
-    "lint": "biome check src/* --write --unsafe",
-    "linter": "biome ci src/*",
-    "lint:check": "biome ci src/*",
+    "lint:fix": "biome check src/* --write --unsafe",
+    "lint": "biome ci src/*",
     "format:check": "biome format src/*",
     "format": "biome format src/* --write",
     "load-git-hooks": "cd .. && git config core.hooksPath \"./.git-hooks\"",
-    "analyze": "RSDOCTOR=true npm run build"
+    "analyze": "RSDOCTOR=true npm run build",
+    "postinstall": "node scripts/fix-remark-emoji.js"
```

Comment:
We need the `remark-emoji` package to be installed with the correct version, hence the `postinstall` script.